zulipterminal/ui_tools/views.py

```diff
@@ -806,6 +806,29 @@ def __init__(self, controller: Any,
         super().__init__(controller, widgets, 'GO_BACK')
 
 
+class AboutView(PopUpView):
+    def __init__(self, controller: Any, ZT_VERSION: str) -> None:
+        content = [
+            ('Zulip Terminal version', ZT_VERSION),
+        ]
+
+        widths = [(len(label), len(data)) for label, data in content]
+        max_widths = [max(width) for width in zip(*widths)]
+        self.width = sum(max_widths) + 2  # Add +2 to recoup `dividechars=2`.
+
+        widgets = []
+        for index, (label, data) in enumerate(content):
+            widgets.append(urwid.AttrWrap(
+                urwid.Columns([(max_widths[0], urwid.Text(label)),
+                               (max_widths[1], urwid.Text(data))],
+                              dividechars=2),
+                None if index % 2 == 0 else 'popup_contrast'
+            ))
```

Comment:
In this commit we don't have more than one line, so including this alternating color code in this first commit is a bit confusing? Also the other multi-line capabilities. This may be less relevant if we ended up merging the other PR first.